app/models/incident.rb

```diff
@@ -163,9 +163,6 @@ def export_to_claims(current_user)
     if completed? && valid?
       begin
         export_claims_xml(current_user)
-        ci = ClaimsIncident.create claims_fields table: :incident
-        update claims_id: ci.UID
-        ClaimsDriversReport.create claims_fields table: :drivers_report
         self.update exported_to_claims: true
         return { status: :success }
       rescue ActiveRecord::StatementInvalid => e
```

Comment:
Probably don't need this `rescue` anymore

Reply:
And, if you remove it, you can remove `#claims_export_error` from `ApplicationMailer`.